setup.py

```diff
@@ -25,7 +25,9 @@
 import warnings
 import sysconfig
 from distutils.version import LooseVersion
+from tools.version_utils import write_version_py
 
+IS_RELEASE_BRANCH = False
```

Comment:
This looks like it should come from `version_utils.py` too.

Reply:
Fixed now!

Reply:
It should be fixed now. Apologies :sweat_smile: